packages/regal/.fvmrc

```diff
@@ -0,0 +1,3 @@
+{
+  "flutter": "3.19.5"
```

Comment:
@alphapokedex why is this getting added?

Reply:
to support fvm